packages/next/src/build/index.ts

```diff
@@ -2489,15 +2504,10 @@ export default async function build(
             ssgPages,
             additionalSsgPaths
           )
-          const exportApp: typeof import('../export').default =
-            require('../export').default
+          const exportApp: ExportAppWorker = require('../export').default
 
           const exportConfig: NextConfigComplete = {
             ...config,
-            initialPageRevalidationMap: {},
-            initialPageMetaMap: {},
-            pageDurationMap: {},
-            ssgNotFoundPaths: [] as string[],
             // Default map will be the collection of automatic statically exported
```

Comment:
It felt really weird that the export path used the configuration as the object for which it wrote data to. Most of the changes in this PR was designed around inverting it so that the `exportApp` simply returned the data from the export, simplifying the flow.